scheduling_strawman.py

```diff
@@ -0,0 +1,99 @@
+"""
+The scheduler is a very useful tool. It requires the input of observing blocks
+and the time_range. Observing blocks consist of a target, a duration and constraints.
+"global" constraints can also be defined and can significantly speed up the scheduler.
+"""
+
+# a few packages are needed to set up the proper objects
+import numpy as np
+from astropy import units as u
+from astropy.coordinates import SkyCoord
+from astropy.time import Time
+import astroplan
+from astroplan.constraints import *
+from astroplan import Observer, FixedTarget
+from astroplan.scheduling import *
+
+# create the time frame during which you observing
+schedule_start = Time.now()+4*u.hour
+schedule_end = schedule_start+24*u.hour
+
+# define the observer
+mro = Observer.at_site('mro')
+
+# we also want to define how quickly the telescope can transition
+# between different targets, filters, instruments etc.
+# for using a telescope that slews at 1 degree/second,
+# takes 2 minutes to change filters, and 10 to change instruments
+slew_rate = 1*u.deg/u*second
+transitioner = Transitioner(slew_rate, {filter: 2*u.minute, instrument: 10*u.minute})
```

Comment:
That seems good. I was wondering how we could handle different types of the same transition, and this would be a good answer.